rslearn/main.py

```diff
@@ -548,10 +552,17 @@ def dataset_ingest() -> None:
         default="",
         help="List of layers to disable e.g 'layer1,layer2'",
     )
+    parser.add_argument(
+        "--ignore-errors",
```

Comment:
Previously errors were being ignored and it would print a warning to the console. This was introduced because there were a few data sources, especially the commercial imagery ones like Planet Labs images, where some items would either have permanent or transient errors ingesting. In the case of transient errors, ignoring the error would allow the current ingestion to continue going through all the items in parallel, and then the user could run ingestion again to try to complete ingesting the ones that failed the first time.

Now that it is being run automatically in inference jobs, it should trigger an error to terminate those jobs, and it now seems better to trigger error by default. But it still seems useful to have an `--ignore-errors` option in case the user is encountering some kind of transient error.